libraries/abstractions/transport/utest/CMakeLists.txt

```diff
@@ -0,0 +1,101 @@
+    project ("transportsecure sockets unit test")
+    cmake_minimum_required (VERSION 3.13)
+
+# ====================  Define your project name (edit) ========================
+    set(project_name "transport_secure_sockets")
+
+# =====================  Create your mock here  (edit)  ========================
+
+# list the files to mock here
+    list(APPEND mock_list
+                "${kernel_dir}/include/task.h"
+                "${kernel_dir}/include/portable.h"
+                "${common_dir}/include/iot_logging_task.h"
```

Comment:
Instead of mocking the `iot_logging_task.h`, `portable.h` and `task.h` files, we can disable logging by defining mock header files for `logging_levels.h` and `logging_stack.h` files.

Reply:
copied logging-stack from CSDK in 388cb8a73993ee3ac039fa574771189be32f60e5